google-cloud-logging/lib/google/cloud/logging/rails.rb

```diff
@@ -63,14 +63,15 @@ def self.init_middleware app
           resource_type = Logging.configure.monitored_resource.type
           resource_labels = Logging.configure.monitored_resource.labels
           log_name = Logging.configure.log_name
+          labels = Logging.configure.labels
```

Comment:
Please add an entry in the [configuration instruction doc](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/stackdriver/docs/configuration.md) for this new configuration option.